docs/graphcache/normalized-caching.md

```diff
@@ -62,6 +62,29 @@ Multiple results may refer to the same piece of data — a mutation for instance
 later on in the app. This would automatically cause _Graphcache_ to update any related queries in
 the entire app, because all references to each each entity are shared.
 
+## Terminology
+
+A few terms that will be used throughout the GraphCache documentation that are important to know
+to get a full understanding.
+
+### Entity
```

Comment:
Does this look good in the markdown document or should this be a list with emphasis marks?